libs/base/ui/structure/heading/src/typography.controller.ts

```diff
@@ -46,7 +46,9 @@ export class TypographyController implements ReactiveController {
    * Sets the font properties for the heading, based on the provided tag and size.
    */
   protected setFont(tag?: HeadingTag | HeadingVisibility, size?: Size): void {
-    if (!tag) return;
+    if (!tag) {
```

Comment:
What was wrong with prev version?

Reply:
nothing, left this as part of refactor. rolling back